AssetEditor/Views/MainWindow.xaml.cs

```diff
@@ -14,14 +16,19 @@ public partial class MainWindow : Window
         Point _lastMouseDown;
         IEditorViewModel draggedItem;
 
+
+
         public MainWindow()
         {
             InitializeComponent();
 
-            System.Reflection.Assembly assembly = System.Reflection.Assembly.GetExecutingAssembly();
-            var fvi = System.Diagnostics.FileVersionInfo.GetVersionInfo(assembly.Location);
+            
+
+            //System.Reflection.Assembly assembly = System.Reflection.Assembly.GetExecutingAssembly();
+            FileVersionInfo fvi = FileVersionInfo.GetVersionInfo(System.Diagnostics.Process.GetCurrentProcess().MainModule.FileName);
+
+            Title = $"{fvi.ProductName} - {fvi.FileVersion}";
 
-            Title = $"AssetEditor v{fvi.FileMajorPart}.{fvi.FileMinorPart}";
```

Comment:
Revert to the old commit to avoid any confusion with older userbase. also for historical preservation too

Reply:
Ok, revert it all the back ole's origina?
Can we do "build numbers":)?

Reply:
well, git doesn't have "revision number" like SVN does. but we can include build commit. here's how I achieved that https://github.com/admiralnelson/bannerlord-error-detector/blob/50f213a6f9418697fb638132d5a0e170727fbc06/Util.vb#L385   
Basically there's inert file called commit.txt which contains the latest commit hash, this file will be generated automatically by visual studio everytime we hit compile button.

Reply:
Update from discord convo: it should be called AssEd45 (45 because we started on version 45) AssEd (because we want to keep it generic and open, also it sounds kinda funny)
[Resolve this and then I can merge it]

Reply:
Done.